app/main/services/query_builder.py

```diff
@@ -108,7 +116,10 @@ def multi_match_clause(mapping, keywords):
     return {
         "simple_query_string": {
             "query": keywords,
-            "fields": mapping.text_fields,
+            "fields": [
+                "_".join((mapping.text_search_field_prefix, field_name))
```

Comment:
This pattern seems to be repeated a lot. Is there any value in abstracting it to its own function?

Reply:
Maybe, but it's only ~10 characters more than the bare minimum of the two argument names in question...